framework/source/class/qx/ui/basic/Image.js

```diff
@@ -780,7 +876,36 @@ qx.Class.define("qx.ui.basic.Image",
      * @param source {String} source path
      */
     __setSource: function (el, source) {
-      if (el.getNodeName() == "div") {
+      var isFont = source && qx.lang.String.startsWith(source, "@");
+
+      if (isFont) {
+        var ResourceManager = qx.util.ResourceManager.getInstance();
+        var font = qx.theme.manager.Font.getInstance().resolve(source.match(/@([^/]+)/)[1]);
+        el.setStyles(font.getStyles());
+        el.setStyle("display", "table-cell");
+        el.setStyle("vertical-align", "middle");
+        el.setStyle("text-align", "center");
+
+        if (this.getScale()) {
+          el.setStyle("font-size", (this.__width > this.__height ? this.__height : this.__width) + "px");
+        }
```

Comment:
Also width/height must be set here, on order to override the values set by  `el.setStyles(font.getStyles())` in line 884